packages/sdk-react/src/components/receivables/InvoiceDetails/ExistingInvoiceDetails/components/OverviewTabPanel.tsx

```diff
@@ -75,6 +90,53 @@ export const OverviewTabPanel = ({
       }
     );
 
+  const { data: receivable } = useReceivableById(invoice.id);
+
+  const creditNoteIds =
+    receivable?.type === 'invoice'
+      ? receivable.related_documents.credit_note_ids
+      : [];
+
+  const {
+    data: creditNoteQuery,
+    isLoading: isCreditNoteLoading,
+    error: creditNoteError,
+  } = useReceivables({
+    id__in: creditNoteIds,
+    type: 'credit_note',
+  });
+
+  const transformCreditNotes = (
+    creditNotes: components['schemas']['InvoiceResponsePayload'][]
+  ): TransformCreditNotes[] => {
+    if (!creditNotes) return [];
+
+    return creditNotes.map((creditNote) => {
+      const issueDate = creditNote.issue_date
+        ? new Date(creditNote.issue_date)
+        : null;
+
+      const formattedDate = issueDate
+        ? issueDate.toLocaleDateString('en-GB').replace(/\//g, '.')
+        : t(i18n)`Unknown date`;
+
+      const authorName =
+        creditNote.entity.type !== 'individual' && creditNote.entity.name
+          ? creditNote.entity.name
+          : creditNote.entity.type !== 'organization'
+          ? `${creditNote.entity.first_name} ${creditNote.entity.last_name}`
+          : null;
+
+      return {
+        title: creditNote.document_id,
+        description: `${t(i18n)`Issued on`} ${formattedDate} ${t(i18n)`by`}`,
+        authorTitle: authorName || '',
+        onClick: () =>
```

Comment:
We should not add any functional _placeholders_ to the `dev`. `dev` is a stable branch. You can use [Feature toggle](https://en.wikipedia.org/wiki/Feature_toggle) in some way if you need to merge changes which are not ready for production.